src/tasks/dump.ts

```diff
@@ -0,0 +1,567 @@
+import readline from "readline";
+
+import chalk from "chalk";
+import {
+  BigNumber,
+  constants,
+  ContractTransaction,
+  Signer,
+  utils,
+} from "ethers";
+import { task, types } from "hardhat/config";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { estimateTradeAmount, getFee, placeOrder } from "../services/api";
+import {
+  BUY_ETH_ADDRESS,
+  domain,
+  Order,
+  OrderKind,
+  SigningScheme,
+  signOrder,
+  TypedDataDomain,
+} from "../ts";
+
+import {
+  getDeployedContract,
+  isSupportedNetwork,
+  SupportedNetwork,
+} from "./ts/deployment";
+import { promiseAllWithRateLimit } from "./ts/rate_limits";
+import { Align, displayTable } from "./ts/table";
+import {
+  isNativeToken,
+  nativeToken,
+  NativeToken,
+  tokenDetails,
+  TokenDetails,
```

Comment:
I guess one of these is a method and one is a type. Is there some sort of TS standard where types are declared in types files so to avoid these kind of imports or am I imagining things?